crates/sound/src/device.rs

```diff
@@ -372,10 +373,11 @@ impl VhostUserSoundThread {
         Ok(false)
     }
 
-    fn process_tx(
+    fn process_io(
         &self,
         vring: &VringRwLock,
         audio_backend: &RwLock<Box<dyn AudioBackend + Send + Sync>>,
+        write: bool,
```

Comment:
What about using an enum?

Reply:
Would certainly be more comprehensible, will do

Reply:
Added a `Direction` enum